doc/source/ray-more-libs/using-ray-with-pytorch-lightning.rst

```diff
@@ -1,6 +1,13 @@
+:orphan:
```

Comment:
Should we just delete most of the text on this page then, and just leave the note?

Reply:
Sorry I just missed this comment. So we can still keep this doc in the drop-down menu, but only with this note?

Reply:
Users can check out the docs in previous release if they are still using ray_lightning.